editor/src/components/canvas/canvas-strategies/absolute-reparent-strategy.tsx

```diff
@@ -30,7 +32,19 @@ export const absoluteReparentStrategy: CanvasStrategy = {
       return filteredSelectedElements.every((element) => {
         const elementMetadata = MetadataUtils.findElementByElementPath(metadata, element)
 
-        return elementMetadata?.specialSizeMeasurements.position === 'absolute'
+        const referencesExternalValue =
+          elementMetadata == null
```

Comment:
Should this strategy just return `false` if there is no metadata? I know it will implicitly because of the `position` check, but I'm wondering if we should make some form a deliberate decision here.